internal/cli/organizations.go

```diff
@@ -461,38 +472,175 @@ auth0 orgs members ls <id>`,
 				inputs.ID = args[0]
 			}
 
-			list, err := getWithPagination(
-				cmd.Context(),
-				inputs.Number,
-				func(opts ...management.RequestOption) (result []interface{}, hasNext bool, apiErr error) {
-					members, apiErr := cli.api.Organization.Members(url.PathEscape(inputs.ID), opts...)
-					if apiErr != nil {
-						return nil, false, apiErr
+			members, err := cli.getOrgMembersWithSpinner(cmd.Context(), inputs.ID, inputs.Number)
+			if err != nil {
+				return err
+			}
+			sortMembers(members)
+			cli.renderer.MembersList(members)
+			return nil
+		},
+	}
+	return cmd
+}
+
+func rolesOrganizationCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "roles",
+		Short: "Manage roles of an organization",
+		Long:  "Manage roles of an organization.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listRolesOrganizationCmd(cli))
+	cmd.AddCommand(membersRolesOrganizationCmd(cli))
+
+	return cmd
+}
+
+func listRolesOrganizationCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		OrgID  string
+		Number int
+	}
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "List roles of an organization",
+		Long:    "List roles assigned to members of an organization.",
+		Example: `auth0 orgs roles list
+auth0 orgs roles ls <id>`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := organizationID.Pick(cmd, &inputs.OrgID, cli.organizationPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.OrgID = args[0]
+			}
+
+			members, err := cli.getOrgMembersWithSpinner(cmd.Context(), inputs.OrgID, inputs.Number)
+			if err != nil {
+				return err
+			}
+
+			roleMap := make(map[string]management.OrganizationMemberRole)
+			err = ansi.Spinner("Getting roles for each member", func() error {
+				for _, member := range members {
+					userID := member.GetUserID()
+					roleList, errInner := cli.api.Organization.MemberRoles(inputs.OrgID, userID)
+					if errInner != nil {
+						return errInner
 					}
-					var output []interface{}
-					for _, member := range members.Members {
-						output = append(output, member)
+					for _, role := range roleList.Roles {
+						roleID := role.GetID()
+						if _, exists := roleMap[roleID]; !exists {
+							roleMap[roleID] = role
+						}
 					}
-					return output, members.HasNext(), nil
-				})
-
+				}
+				return nil
+			})
```

Comment:
Would it be possible to abstract this within a func?:) Similar to what we did for `cli.getOrgMembersWithSpinner()` above?

e.g.
```go
memberRoleMap, err := cli.getOrgMemberRolesWithSpinner(...)
```